settings.go

```diff
@@ -29,64 +29,64 @@ func (h *MultiOption) Set(value string) error {
 
 // AppSettings is the struct of main configuration
 type AppSettings struct {
-	verbose   bool
-	debug     bool
-	stats     bool
-	exitAfter time.Duration
-
-	splitOutput          bool
-	recognizeTCPSessions bool
-	pprof                string
-
-	inputDummy   MultiOption
-	outputDummy  MultiOption
-	outputStdout bool
-	outputNull   bool
-
-	inputTCP        MultiOption
-	inputTCPConfig  TCPInputConfig
-	outputTCP       MultiOption
-	outputTCPConfig TCPOutputConfig
-	outputTCPStats  bool
-
-	inputFile        MultiOption
-	inputFileLoop    bool
-	outputFile       MultiOption
-	outputFileConfig FileOutputConfig
-
-	inputRAW                MultiOption
-	inputRAWEngine          string
-	inputRAWTrackResponse   bool
-	inputRAWRealIPHeader    string
-	inputRAWExpire          time.Duration
-	inputRAWProtocol        string
-	inputRAWBpfFilter       string
-	inputRAWTimestampType   string
+	Verbose   bool          `json:"verbose"`
+	Debug     bool          `json:"debug"`
+	Stats     bool          `json:"stats"`
+	ExitAfter time.Duration `json:"exit-after"`
+
+	SplitOutput          bool   `json:"split-output"`
+	RecognizeTCPSessions bool   `json:"recognize-tcp-sessions"`
+	Pprof                string `json:"http-pprof"`
+
+	InputDummy   MultiOption `json:"input-dummy"`
+	OutputDummy  MultiOption `json:"output-dummy"`
+	OutputStdout bool        `json:"output-stdout"`
+	OutputNull   bool        `json:"output-null"`
+
+	InputTCP        MultiOption     `json:"input-tcp"`
+	InputTCPConfig  TCPInputConfig  `json:"input-tcp"`
```

Comment:
two keys have input-tcp json key, is it correct?

Reply:
Fixed it.